plugin-admin-notes.php

```diff
@@ -452,16 +452,12 @@ function wds_plugin_police() {
  * @since 0.1.0
  */
 function uninstall_wds_plugin_admin_notes() {
-	$args = array(
-		'post_type'      => 'wdspp_plugin_police',
-		'fields'         => 'id',
-		'cache_results'  => false,
-		'posts_per_page' => - 1,
-	);
 
-	$query = new WP_Query( $args );
+	global $wpdb;
 
-	foreach ( $query->posts as $post ) {
+	$ids = $wpdb->get_col( "SELECT ID FROM $wpdb->posts WHERE post_type='wdspp-plugin-police'" );
+
+	foreach ( $ids as $post ) {
```

Comment:
hey @binarygary - what about checking to make sure posts were found & bailing if not before hitting this foreach? (sorry, I think I merged this PR before noticing that)